src/GitHub.App/Services/UsageTracker.cs

```diff
@@ -0,0 +1,378 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.Diagnostics;
+using System.Globalization;
+using System.Linq;
+using System.Reactive;
+using System.Reactive.Linq;
+using Akavache;
+using GitHub.Caches;
+using GitHub.Extensions.Reactive;
+using GitHub.Models;
+using GitHub.Settings;
+using ReactiveUI;
+using Rothko;
+
+#pragma warning disable CS0649
+
+namespace GitHub.Services
+{
+    [Export(typeof(IUsageTracker))]
+    [PartCreationPolicy(CreationPolicy.Shared)]
+    public class UsageTracker : IUsageTracker
+    {
+        // Whenever you add a counter make sure it gets added to _both_
+        // BuildUsageModel and ClearCounters
```

Comment:
Yeah, this came from the GHfW codebase - the changes in your PR looks good to me.